docs/add-a-benchmark.rst

```diff
@@ -7,27 +7,34 @@
 Adding a Benchmark
 ==================
 
-The following system-independent specification is required for each benchmark: 
+The following system-independent specification is required for each benchmark:
 
 - ``package.py`` is a Spack specification that defines how to build and install the benchmark.
 - ``application.py`` is a Ramble specification that defines the benchmark input and parameters.
 
 By default, Benchpark will use the benchmark specifications provided in the Spack and Ramble repos.
 
+==============
 benchpark/repo
---------------
-If  you are working on a benchmark that is not (yet) available in Spack and/or Ramble,
-or you would like to override the benchmark specifications available in Spack and/or Ramble,
-you can add the following specification in the ``benchpark/repo``::
+==============
+Benchpark provides a staging area for the build and run specification for a benchmark
+so that you can test them together with your experiment specification on a given system::
 
   benchpark
-  └── repo 
-     ├── ${BENCHMARK1} 
-     │  ├── application.py 
-     │  └── package.py 
-     └── repo.yaml 
+  └── repo
+     ├── ${Benchmark1}
```

Comment:
(recommendation) Might be useful to tie this to an example invocation like

```
benchpark setup ${Benchmark1}/${ProgrammingModel1} ${System1} /output/path/to/experiments_root
```

overall, I think this specification will be clearest if it leads by discussing that you can override Ramble/Spack descriptions; the rewording indicates to me that you're trying to avoid that, but to manipulate these at all, the user will have to be familiar with them anyhow.